samples/net/echo_server/overlay-tcp2.conf

```diff
@@ -0,0 +1,34 @@
+
+CONFIG_NET_TCP2=y
+
+CONFIG_NET_PKT_RX_COUNT=1024
+CONFIG_NET_PKT_TX_COUNT=1024
+CONFIG_NET_BUF_RX_COUNT=1024
+CONFIG_NET_BUF_TX_COUNT=1024
+
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="10.0.0.1"
```

Comment:
Zephyr standardized on 192.0.2.1/192.0.2.2 addresses for these. Using something else leads to additional mental effort in dealing with and potential confusion. The same problem IIRC applies to your TTCN suite setup. Please update to use the addresses above.

Reply:
> Zephyr standardized on 192.0.2.1/192.0.2.2 addresses for these.

I think "standardized" is too strong wording here. The 192.0.2/24 addresses are reserved for documentation only, and cannot be used in real products thus the values were selected in sample project files. But I agree in general, that using 192.0.2/24 address space would be better here.